src/sharedInterfaces/queryResult.ts

```diff
@@ -26,6 +26,7 @@ export interface QueryResultWebviewState {
     uri?: string;
     resultSetSummary?: ResultSetSummary;
     value?: string;
+    theme?: string;
```

Comment:
theme is also added to `QueryResultState`, right? We can just use that one and avoid adding theme property here.

Reply:
Yeah, I just updated this